frontend/packages/schema-model/src/lib/SchemaModel.ts

```diff
@@ -432,9 +438,10 @@ export class SchemaModel {
   }
 
   public deleteNode(pointer: string): SchemaModel {
-    if (pointer === ROOT_POINTER) throw new Error('It is not possible to delete the root node.');
-    if (this.isDefinitionInUse(pointer))
-      throw new Error('Cannot delete a definition that is in use.');
+    if (pointer === ROOT_POINTER) {
+      throw new Error('It is not possible to delete the root node.');
```

Comment:
Should this be added to the translation file? 🤔

Reply:
It's not visible for users, but it gets logged to application insights. ☺️